crates/cheatnet/tests/cheatcodes/mod.rs

```diff
@@ -5,3 +5,5 @@ mod precalculate_address;
 mod roll;
 mod spy_events;
 mod warp;
+
+mod forking;
```

Comment:
1. Forking should be moved to a different testing module or at least integrated with the starknet module as it is not a cheatcode from the Cheatnet's perspective and we should be able to run it's test separately.